app/javascript/packages/verify-flow/services/api.ts

```diff
@@ -1,4 +1,4 @@
-export interface ErrorResponse<Field extends string> {
+export interface ErrorResponse<Field extends string = string> {
```

Comment:
is the `= string` providing like a default field type, so we could just do `ErrorResponse` instead of needing to specify `ErrorResponse<'myfield'>`?

Reply:
> is the `= string` providing like a default field type, so we could just do `ErrorResponse` instead of needing to specify `ErrorResponse<'myfield'>`?

Yep, exactly. Used for the password reset response elsewhere in this branch:

https://github.com/18F/identity-idp/blob/14310ed2e498bc31eb32493b4afd437d94c29f66/app/javascript/packages/verify-flow/steps/password-confirm/password-reset-button.tsx#L21